addon/services/page-title-list.js

```diff
@@ -16,6 +16,9 @@ const RouterEvent = {
   @extends Ember.Service
  */
 export default class PageTitleListService extends Service {
+  @service
```

Comment:
This way the new `page-title` service is initialised when the `page-title-list` service is initialised. Otherwise simply extending the `page-title` service in the host app has no effect and the user needs to do this manually somehow.

Reply:
This is fine.